doc/smtp.md

```diff
@@ -0,0 +1,13 @@
+# Cloud Resource Operator - SMTP 
+
+## Usage
+To seed a Kubernetes/Openshift cluster with an example SMTP resource:
+```
+$ make cluster/prepare 
+$ make cluster/seed/smtp
+```
+
+## Strategy 
+A config map object is expected to exist for each provider that will be used by the operator. This config map contains provider-specific information about how to deploy a particular resource type, such as blob storage. In the Cloud Resources Operator, this provider-specific configuration is called a strategy. An example of the AWS strategy can be seen [here](../deploy/examples/config_aws_strat.yaml)
+### AWS Strategy
+A JSON object containing two keys - `region`, which is a supported [AWS region code](https://docs.aws.amazon.com/general/latest/gr/rande.html#ses_region). `strategy`, which for now does not have any options
```

Comment:
Can you add the AWS region code link to the others as well? Also do we need to include strategy if it's not supported yet?